test_rmw_implementation/test/test_publisher.cpp

```diff
@@ -375,3 +375,27 @@ TEST_F(CLASSNAME(TestPublisherUse, RMW_IMPLEMENTATION), count_mismatched_subscri
   EXPECT_EQ(RMW_RET_OK, ret) << rmw_get_error_string().str;
   EXPECT_EQ(0u, subscription_count);
 }
+
+TEST_F(CLASSNAME(TestPublisherUse, RMW_IMPLEMENTATION), publish_message_bad_arguments) {
+  test_msgs__msg__BasicTypes input_message{};
+  ASSERT_TRUE(test_msgs__msg__BasicTypes__init(&input_message));
+  // const rosidl_runtime_c__Sequence__bound * message_bounds = ROSIDL_GET_SEQUENCE_BOUNDS(test_msgs, msg, BasicTypes);
+  rmw_publisher_allocation_t * default_allocation{nullptr};
+  // rmw_ret_t ret = rmw_init_publisher_allocation(ts, message_bounds, default_allocation);
+  void * msg_pointer = &input_message;
```

Comment:
@Lobotuerk why not using `&input_message` directly elsewhere?

Reply:
Makes sense